automated-comments/go/two-fer/work_with_any_provided_name.md

```diff
@@ -0,0 +1,2 @@
+This solution references the names `Alice` and `Bob` directly in the code. This is a good first step!
```

Comment:
```suggestion
Change the function to work with any name provided. Currently the solution only works with `Alice` and `Bob`, which is a first good step.
```

I would prefer to drop line 2, and have this be actionable.

Reply:
```suggestion
```